### PR TITLE
chat list update doc

### PR DESCRIPTION
**Pull request by bhartono**

### CODE REVIEW

File: api-reference/beta/api/chat-list.md [hunk 1]
```diff
@@ -429,6 +429,59 @@ Content-type: application/json
...
+### Example 4: List all the chats with certain number of item per page
+
+#### Request
+
+Here is an example of the request.
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "list_chats"
```
Comment (AkJo) [inline api-reference/beta/api/chat-list.md:441]: This needs to be unique across requests.

list_chats_with_pagination

File: api-reference/beta/api/chat-list.md [hunk 2]
```diff
@@ -429,6 +429,59 @@ Content-type: application/json
 }
 ```
 
+### Example 4: List all the chats with certain number of item per page
```
Comment (AkJo) [inline api-reference/beta/api/chat-list.md:432]: List all chats, two at a time.

File: api-reference/v1.0/api/chat-list.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+### Example 1: List all the chats
+
+#### Request
+
+Here is an example of the request.
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "list_chats"
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:64]: list_user_chats

File: api-reference/v1.0/api/chat-list.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+
+## Example
+
+### Example 1: List all the chats
+
+#### Request
+
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:61]: Remove.

File: api-reference/v1.0/api/chat-list.md [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+}
+```
+
+### Example 4: List all the chats with certain number of item per page
+
+#### Request
+
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:382]: Please remove.

File: api-reference/v1.0/api/chat-list.md [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "list_chats"
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:385]: Please update based on beta

File: api-reference/v1.0/api/chat-list.md (hunk 5 above)
Comment (AkJo) [reply api-reference/v1.0/api/chat-list.md:382]: That head is autogenerated when the snippet code block is processed.

File: api-reference/v1.0/api/chat-list.md [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+}
+```
+
+### Example 3: List all the chats that have a member with a specific display name
+
+#### Request
+
+Here is an example of a request that will filter all the chats based on a specific member's display name.
+
+# [HTTP](#tab/http)
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:277]: Please remove.

File: api-reference/v1.0/api/chat-list.md [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+                    "userId": "d74fc2ed-cb0e-4288-a219-b5c71abaf2aa",
+                    "email": "parkerp@teamsgraph.onmicrosoft.com"
+                }
+            ]
+        }
+    ]
+}
+```
+
+### Example 4: List all the chats with certain number of item per page
```
Comment (AkJo) [inline api-reference/v1.0/api/chat-list.md:376]: Perhaps rephrase to:
List all chats with not more than a specific number of chats in the response.

@Lauragra, suggestions?

File: api-reference/v1.0/toc.yml [hunk 9]
```diff
@@ -431,6 +431,8 @@ items:
           href: api/userscopeteamsappinstallation-get-chat.md 
         - name: Get chat for user
           href: api/chat-get.md
+        - name: List chats for user
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:434]: Does this need to be exposed under Teamwork/chat in the TOC?

File: changelog/Microsoft.Teams.Core.json [hunk 10]
```diff
@@ -1,6 +1,23 @@
...
-
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "ecb01663-dbaa-4564-ba93-18a354c750aa",
+          "ApiChange": "Method",
+          "ChangedApiName": "chat",
+          "ChangeType": "Addition",
+          "Description": "Added the [List chat members](https://docs.microsoft.com/en-us/graph/api/chat-list-members?view=graph-rest-1.0&preserve-view=true) method.",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:10]: We're adding the List chats operation, right?

**Pull request closed without merging**